src/pages/Equipments.vue

```diff
@@ -0,0 +1,307 @@
+              </div>
+              <div class="flex-row">
+                <EquipmentsIcon 
+                  v-bind:equipmentType="EquipmentType.Shoes"
+                  v-bind:equipmentId="'shoes'"
+                  @clicked="equipmentSelected"
+                />
+                <EquipmentsIcon 
+                  v-bind:equipmentType="EquipmentType.Ring"
+                  v-bind:equipmentId="'ring2'"
+                  @clicked="equipmentSelected"
+                />
+              </div>
+            </div>
+          </div>
+          <div v-if="currentTab === 1">
+            <div class="flex-column">
+              <div class="flex-row">
+                <EquipmentsIcon 
+                  v-bind:equipmentType="EquipmentType.Pickaxe"
+                  v-bind:equipmentId="'pickaxe'"
+                  @clicked="equipmentSelected"
+                />
+                <EquipmentsIcon 
+                  v-bind:equipmentType="EquipmentType.Axe"
+                  v-bind:equipmentId="'axe'"
+                  @clicked="equipmentSelected"
+                />
+              </div>
+              <div class="flex-row">
+                <EquipmentsIcon 
+                  v-bind:equipmentType="EquipmentType.FishingRod"
+                  v-bind:equipmentId="'fishingrod'"
+                  @clicked="equipmentSelected"
+                />
+                <EquipmentsIcon 
+                  v-bind:equipmentType="EquipmentType.CatchingNet"
+                  v-bind:equipmentId="'catchingnet'"
+                  @clicked="equipmentSelected"
+                />
+              </div>
+              <div class="flex-row">
+                <EquipmentsIcon 
+                  v-bind:equipmentType="EquipmentType.TrapBox"
+                  v-bind:equipmentId="'trapbox'"
+                  @clicked="equipmentSelected"
+                />
+                <EquipmentsIcon 
+                  v-bind:equipmentType="EquipmentType.WorshipSkull"
+                  v-bind:equipmentId="'worshipskull'"
+                  @clicked="equipmentSelected"
+                />
+              </div>
+              <div class="flex-row">
+                <EquipmentsIcon 
+                  v-bind:equipmentType="EquipmentType.FishingLine"
+                  v-bind:equipmentId="'fishingline'"
+                  @clicked="equipmentSelected"
+                />
+                <EquipmentsIcon 
+                  v-bind:equipmentType="EquipmentType.FishingWeight"
+                  v-bind:equipmentId="'fishingweight'"
+                  @clicked="equipmentSelected"
+                />
+              </div>
+            </div>
+          </div>
+          <div v-if="currentTab === 2">
+            <div class="flex-column">
+              <div class="flex-row">
+                <EquipmentsIcon 
+                  v-bind:equipmentType="EquipmentType.Food"
+                  v-bind:equipmentId="'food1'"
+                  @clicked="equipmentSelected"
+                />
```

Comment:
Same thing here. Consider a different component/implementation for food since it can't be upgraded/have different base stats, just bonus multipliers.